app/src/main/res/values/strings.xml

```diff
@@ -35,7 +35,7 @@
     <string name="developer_options_events">Input Events</string>
     <string name="developer_options_display_density">Display Density:</string>
     <string name="developer_options_window_size">Window Size:</string>
-    <string name="developer_options_window_size_by">by</string>
+    <string name="developer_options_by">By</string>
```

Comment:
the L10n string should be like `Window Size: {width} × {height}` or `Window Size: {width} by {height}`

we can't use the word `By` by itself, because the ordering of words doesn't make sense in all other languages